exercises/pythagorean-triplet/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "java"
+apply plugin: "eclipse"
+apply plugin: "idea"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testCompile "junit:junit:4.12"
+  testCompile "org.assertj:assertj-core:3.2.0"
```

Comment:
Let's pop this bad-boy (i.e. `assertj`) out as we're aiming to minimize the number of 3rd party dependencies (very recent thing).  ("policy" at exercism/exercism.io#3113).